app/src/main/java/com/cornellappdev/android/eatery/ui/screens/SearchScreen.kt

```diff
@@ -159,7 +159,18 @@ fun SearchScreen(
                                     } else {
                                         searchViewModel.addFilter(filter)
                                     }
-                                })
+                                },
+                                filters = listOf(
+                                    Filter.FromEatery.North,
+                                    Filter.FromEatery.West,
+                                    Filter.FromEatery.Central,
+                                    Filter.FromEatery.BRB,
+                                    Filter.FromEatery.Swipes,
+                                    Filter.RequiresFavoriteEateries.Favorites,
+                                    Filter.CustomFilter.Selected,
+                                    Filter.FromEatery.Under10
+                                )
+                            )
```

Comment:
Ya so here let's definitely pull this out into the viewmodel and just read it from there.